source/install/cloud-changelog.md

```diff
@@ -4,12 +4,78 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 - [Release 2022-05-26](#release-2022-05-26)
 - [Release 2022-05-12](#release-2022-05-12)
 - [Release 2022-04-28](#release-2022-04-28)
 - [Release 2022-04-13](#release-2022-04-13)
 - [Release 2022-03-30](#release-2022-03-30)
-- [Release 2022-03-16](#release-2022-03-16)
+
+## Release 2022-06-15
+
+### Highlights
+
+#### Free Forever Mattermost Cloud Plan
+ - Mattermost Cloud now supports a free forever plan, Cloud Starter, for unlimited users for unlimited time.
+ - Access unlimited channels, playbooks, and boards across the Mattermost platform, with unlimited voice calls and screen sharing in Direct Messages.
+ - The free plan includes a few workspace limits. [Upgrade to Professional for unlimited access](https://mattermost.com/pricing/):
+    - Ability to start a free 30-day Enterprise trial, with access to all features during the trial period.
+    - Maximum 1 team. Additional teams can be created during a trial or on a paid plan, which are archived if downgraded back to Starter. History will be preserved, though that archived history can only be accessed if a free user upgrades to a paid subscription plan.
+    - 10GB file storage across the platform, with 100MB upload limit. History will be preserved.
+    - Unlimited installed apps or plugins, with maximum of 5 enabled at one time. History will be preserved.
+    - Access to 10,000 most recent messages. History will be preserved.
+    - Access to 500 most recently updated cards.
+    - Maximum 5 saved views per board.
+
+#### Calls (Beta)
+ - [Native voice calling and screen sharing](https://docs.mattermost.com/channels/make-calls.html) is now available. This is a Channels-specific integration.
+
+#### Collapsed Reply Threads (General Availability)
+ - [Collapsed Reply Threads](https://docs.mattermost.com/channels/organize-conversations.html) is now generally available. Please reference [this article](https://support.mattermost.com/hc/en-us/articles/6880701948564) for more information and guidance for enabling the feature.
+
+#### Apps Bar (Beta)
+ - The channel header is now decluttered to make it more obvious how to access Calls, Playbooks, and Boards when viewing a channel. All channel header icons registered by plugins are moved to the new Apps Bar, while Calls remains in the channel header.
+
+### Improvements
+
+#### User Interface (UI)
+ - Applied new designs for the Login screen.
+ - Changed some tooltips to appear when focused instead of just on hover.
+ - The legacy **Settings > Advanced Settings > Enable Post Formatting** and **Settings > Advanced Settings > Preview Pre-release Features** settings are now deprecated in favor of the the new formatting toolbar.
+ - Romanian language support was downgraded to Beta.
+
+#### Administration
+ - For new Cloud Workspaces, the default value for **Collapsed Reply Threads** in **System Console > Posts** is now set to **Always On**. Does not impact existing workspaces where this value is already set.
```

Comment:
For all Cloud Workspaces, the default value for **Collapsed Reply Threads** in **System Console > Posts** is now set to **Always On**. You may choose a different [configuration](https://docs.mattermost.com/configure/configuration-settings.html#collapsed-reply-threads) as desired.